hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/interceptors/AutocreateTenantRequestFilter.java

```diff
@@ -46,14 +51,27 @@
 @ServerInterceptor
 public class AutocreateTenantRequestFilter implements ContainerRequestFilter {
 
+    /* list of URI chunks to which this filter should not be applied */
+    private static final List<String> uriExceptions = Arrays.asList(".*/inventory/status/?", ".*/inventory/ping/?",
+            ".*/inventory/?");
+    private static List<Pattern> uriExceptionPatterns;
```

Comment:
should also be `final`. Also, I'd change the def to:
```java
private static final List<Pattern> uriExceptionPatterns = Stream.of(".*/inventory/status/?", ".*/inventory/ping/?", ".*/inventory/?").map(Pattern::compile).collect(Collectors.toList());
```
to save on the one unnecessary field.